projects/mac_ai/toolbox/mac_ai.py

```
@@ -192,6 +196,8 @@ def remote_ramalama_run_bench(
             ngl=99,
             device=None,
             image="quay.io/ramalama/ramalama:latest",
+            oci_runtime=None,
+            pod_env={},
```

Ruff (0.14.5)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)